app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.java

```diff
@@ -121,6 +122,7 @@
 				new AddWheelChairAccessToilets(o),
 				new AddReligionToWaysideShrine(o),
 				new AddBikeParkingType(o),
+				new AddSegregated(o),
```

Comment:
I think what is segregated could be put in the class name here

Reply:
Previously it was baroque AddSegregatedStatusOfCombinedFootwayAndCycleway but it was ridiculous. 

AddIsCyclewaySegregated?

Reply:
AddCyclewaySegregated?

Reply:
AddCyclewaySegregated sounds like it is about mapping cycleways. AddCyclewaySegregatedStatus?

Reply:
AddCyclewaySegregation?

Reply:
that is good, thanks